pkg/cli/image/info/info.go

```diff
@@ -116,6 +136,10 @@ func (o *InfoOptions) Run() error {
 	if err != nil {
 		return err
 	}
+	if len(o.ICSPFile) > 0 || o.ClusterICSP {
+		registryContext = registryContext.WithAlternateBlobSourceStrategy(
+			strategy.NewSimpleLookupICSPStrategy(o.ICSPFile, o.operatorClient.ImageContentSourcePolicies()))
```

Comment:
I'd prefer to have two strategies - one for file (`NewExplicitICSPStrategy`) and one for cluster lookup (`NewICSPFromClusterOnError`).

Reply:
Should both of them have similar behavior or they will behave differently, as in:
- `NewExplicitICSPStrategy` reads always first from alternative (just like Miroslav is saying)
- `NewICSPFromClusterOnError` is similar to current `SimpleLookup` in that it fails first and tries alternates only then?